app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.kt

```diff
@@ -95,7 +89,13 @@ class TaskDetailFragment : Fragment() {
                 }
             }
         }
-        viewDataBinding.setLifecycleOwner(this.viewLifecycleOwner)
+        viewDataBinding.lifecycleOwner = this.viewLifecycleOwner
+
+        val taskId = arguments?.let {
+            TaskDetailFragmentArgs.fromBundle(it).TASKID
```

Comment:
I think you should be using by navArgs: https://developer.android.com/guide/navigation/navigation-pass-data

Reply:
Done